runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpStateSingle.java

```diff
@@ -241,16 +283,16 @@ boolean update(int currentXp, int goalStartXp, int goalEndXp)
 	XpSnapshotSingle snapshot()
 	{
 		return XpSnapshotSingle.builder()
-			.startLevel(Experience.getLevelForXp(startLevelExp))
-			.endLevel(Experience.getLevelForXp(endLevelExp))
-			.xpGainedInSession(xpGained)
-			.xpRemainingToGoal(getXpRemaining())
-			.xpPerHour(getXpHr())
-			.skillProgressToGoal(getSkillProgress())
-			.actionsInSession(actions)
-			.actionsRemainingToGoal(getActionsRemaining())
-			.actionsPerHour(getActionsHr())
-			.timeTillGoal(getTimeTillLevel())
-			.build();
+				.startLevel(Experience.getLevelForXp(startLevelExp))
```

Comment:
Leave the indent as it was before

Reply:
Yeah apparently my continuation indent was set to 8 instead of 4; fixed.